src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.tsx

```diff
@@ -15,58 +15,47 @@
  */
 
 import React from 'react';
-import { Link } from 'react-router-dom';
-import { connect } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 import upperFirst from 'lodash/upperFirst';
+import { Link } from 'react-router-dom';
 
 import analyticsTracking from 'src/services/analytics-service';
 import * as urlFor from 'src/shared/helpers/urlHelper';
 import { buildFocusIdForUrl } from 'src/shared/state/ens-object/ensObjectHelpers';
-import { getBrowserActiveGenomeId } from 'src/content/app/browser/browserSelectors';
+
+import { getActiveGenomeId } from 'src/content/app/species/state/general/speciesGeneralSelectors';
```

Comment:
Nearly missed this one 😞 Discovered it while manually testing locally.
This is the wrong selector. It gets the active genome id on the species page, while we are interested in the active genome id on the browser page. Used on the browser page, it was returning `null`, which broke the app.

I've fixed the selector and updated the tests to check the generated .

Reply:
Thanks for fixing it @azangru